contrib/window_tshirt_sizes/src/sizes.cpp

```diff
@@ -0,0 +1,139 @@
+
+#include <cassert>
+#include <stdio.h>
+
+class SDL_Point {
+public:
+	int x            = 0;
+	int y            = 0;
+	const char* name = nullptr;
+	constexpr SDL_Point(const int _x, const int _y, const char* _name = nullptr)
+	        : x(_x),
+	          y(_y),
+	          name(_name){};
+};
+
+constexpr SDL_Point eight_by_five_ratio = {8, 5, "widescreen"};
+constexpr SDL_Point four_by_three_ratio = {4, 3, "standard"};
+
+constexpr SDL_Point list_of_aspect_ratios[] = {eight_by_five_ratio,
+                                               four_by_three_ratio};
+
+constexpr int small_percent  = 33;
+constexpr int medium_percent = 50;
+constexpr int large_percent  = 75;
+constexpr int full_percent   = 100;
+
+constexpr int list_of_percent_scalers[] = {small_percent,
+                                           medium_percent,
+                                           large_percent,
+                                           full_percent};
+
+// https://en.wikipedia.org/wiki/Display_resolution
+constexpr SDL_Point list_of_screen_dimensions[] = {
+        {640, 480, "VGA"},
+        {720, 480, "480p NTSC"},
+        {726, 576, "480p PAL"},
+        {800, 600, "SVGA"},
+        {1024, 768, "XGA"},
+        {1280, 720, "720p"},
+        {1280, 800, "WXGA"},
+        {1280, 1024, "Super-eXtended Graphics Array (SXGA)"},
+        {1360, 768, "High Definition (HD)"},
+        {1366, 768, "High Definition (HD)"},
+        {1440, 900, "WXGA+"},
+        {1536, 864, "N/A"},
+        {1600, 900, "High Definition Plus (HD+)"},
+        {1680, 1050, "WSXGA+"},
+        {1600, 1200, "High Definition Plus (HD+)"},
+        {1920, 1080, "Full High Definition (FHD)"},
+        {1920, 1200, "Wide Ultra Extended Graphics Array (WUXGA)"},
+        {2048, 872, "Cinemascope"},
+        {2048, 1152, "QWXGA 16:9"},
+        {2048, 1536, "QXGA 4:3"},
+        {2048, 1556, "Film (full-aperture)"},
+        {2560, 1080, "UWFHD roughly 21:9"},
+        {2560, 1440, "Quad High Definition (QHD)"},
+        {2560, 1600, "WQXGA 16:10"},
+        {3440, 1440, "Wide Quad High Definition (WQHD)"},
+        {3840, 2160, "4K or Ultra High Definition (UHD)"},
+        {4096, 3072, "4K reference resolution"},
+        {7680, 4320, "8K"},
```

Comment:
Also, I don't think there's too much value in these names (99% of people would have zero idea what you're talking about if you say FHD, XGA, or QWXGA, etc.)